### DISCUSSION TITLE
Native October CMS e-commerce & the OFFLINE Mall plugin

### DISCUSSION THREAD

**Discussion by chocolata** [OFFLINE-GmbH/oc-mall-plugin#986] [General]
Hi guys,

I noticed the October CMS team is considering creating their own e-commerce solution:
https://portal.octobercms.com/c/42-e-commerce

As the OFFLINE Mall plugin is the most complete e-commerce system I've ever used, imho it would seem redundant for the October CMS team to start from scratch. 

Without being presumptuous - would it be an option to work together with them on this? 
This way we'd have the best of both worlds:

1. We'd keep all the work, all the trial and error that has gone into this plugin (I feel the October CMS team might underestimate this)
2. We'd (hopefully) make use of the additional workforce of the October CMS team to tackle the few things that are still missing from this plugin and guarantee continuity

My fear is that if the October CMS team would go through with their plans on an independent basis, it might interfere with the future of the Mall plugin, causing continuity issues for the clients we've committed to it.

At first glance I see that the October CMS team is thinking about a subscription fee for this new feature, so that might be a more difficult topic considering all the work that has been put into this software.

What are your thoughts about this?

P.S.: I'd be more than willing to pay for the OFFLINE Mall aswel, just to let you know, especially if this would guarantee continuity and further development.

**Comment by damsfx** [comment]
Personally, I think it's a good idea for Offline to continue to offer this plugin under a licence that allows the community to benefit from it free of charge and, above all, to participate in its further development.

I don't really understand the OCMS team's approach in stepping on the toes of an existing free plugin with paid add-ons  [Shopaholic](https://octobercms.com/plugins?search=Shopaholic) ... but I think they have some affinity with Lovata, which develops this plugin.
Do they need money that badly?

It's always a bonus to have a choice
